pandas/core/computation/eval.py

```diff
@@ -80,7 +84,7 @@ def _check_parser(parser):
                        ' {valid}'.format(parser=parser, valid=_parsers.keys()))
 
 
-def _check_resolvers(resolvers):
+def _check_resolvers(resolvers: Iterable) -> None:
```

Comment:
If it's possible we would want to specify the type of object within container classes, so `Iterable[<type>]` if it can be inferred here